Maquetado/scss/detail.scss

```
@@ -32,11 +38,16 @@
 }
 .bookImage {
   width: 250px;
-  height: 360px;
-  margin-right: 25px;
-  margin-left: 20px;
+  height: 385px;
+  padding: 20px 20px 20px 20px
+}
+.textContainer{
+  padding: 20px
 }
 .bookContainer{
   display: flex;
-  margin-top: 30px;
+  margin: 30px;
+  width: 65%;
+  box-shadow: 4px 4px 9px -4px rgba(0,0,0,1);
```

Although `rgba` admits 4 parameters the first 3 can be simplified in an hex color. So in this case you can use the `$black` variable that you already defined.

```suggestion
  box-shadow: 4px 4px 9px -4px rgba($black,1);
```